### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by utam0k**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,23 +26,18 @@ Your ideas are welcome [here](https://github.com/containers/youki/issues/10).
 
 [User Documentation](https://containers.github.io/youki/user/basic_setup.html#quick-install)
 
-# About the name
-
-youki is pronounced as /joʊki/ or yoh-key.
-youki is named after the Japanese word 'youki', which means 'a container'. In Japanese language, youki also means 'cheerful', 'merry', or 'hilarious'.
```
Comment (YJDoc2) [inline README.md:32]: I think we should keep this part, even though not strictly necessary.

File: README.md [hunk 2]
```diff
@@ -322,15 +271,9 @@ cd youki
 just youki-dev # or youki-release
 ```
 
-# Community
-
-We also have an active [Discord](https://discord.gg/h7R3HgWUct) if you'd like to come and chat with us.
```
Comment (YJDoc2) [inline README.md:327]: Maybe we should add the discord link in the docs contributing section, I don't think it is there right now.

**Review by YJDoc2 — COMMENTED** [review]
lgtm, a couple of minor questions

File: README.md (hunk 1 above)
Comment (yihuaf) [reply README.md:32]: I agree :)

File: README.md [hunk 3]
```diff
@@ -93,60 +68,34 @@ Here is why we are writing a new container runtime in Rust.
...
 
   </details>
 
 - I have fun implementing this. In fact, this may be the most important.
 
-# Related project
+# 📍 Status of youki
 
-- [containers/oci-spec-rs](https://github.com/containers/oci-spec-rs) - OCI Runtime and Image Spec in Rust
+youki has aced real-world use cases, including containerd's e2e test, and is now adopted several production environments.
```
Comment (yihuaf) [inline README.md:83]:
```suggestion
**youki** has aced real-world use cases, including containerd's e2e test, and is now adopted by several production environments.
```

**Review by yihuaf — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (utam0k) [reply README.md:32]: okay!

**Comment by utam0k** [comment]
Since this PR only changed README.md, the CI has never started. I'll merge this PR with my privilege.

**Pull request merged**